### DISCUSSION TITLE
Is there a limit to the number of movies PMM can handle?

### DISCUSSION THREAD

**Discussion by WasabiNME** [meisnate12/Plex-Meta-Manager#778] [Q&A]
I made a basic config file to test out on a couple smaller movie libraries only using meisnate12/MovieCharts to populate collections, and everything went swimmingly. But when I pointed it to a 34,000+ movie library, I get this:

```
| Loading All Movies from Library: LQ Movies
| Loaded 34335 Movies
| Mapping Movie Library: LQ Movies
| Traceback (most recent call last):
| File "C:\Users\AMD\AppData\Local\Programs\Python\Python38\lib\site-packages\urllib3\connectionpool.py", line 670, in urlopen | httplib_response = self._make_request(
| File "C:\Users\AMD\AppData\Local\Programs\Python\Python38\lib\site-packages\urllib3\connectionpool.py", line 426, in _make_request | six.raise_from(e, None)
```

etc.

Whenever I broke something earlier during testing, I got actual error messages instead of a crash. Is there a hard-coded limit to how many items can be processed? Or any other ideas? I have no idea how to go about poking around in python.

**Comment by chazlarson** [comment]
There is no hard-coded limit.  My primary movie library has 47000 movies in it, and PMM works with it fine.

It would probably be better to take this sort of thing to the discord.

**Comment by WasabiNME** [comment]
Optimized database and cleaned bundles - problem solved. Works a treat now. Must have had something wacked out in that library.

Cheers!
